packages/mgt-spfx-utils/README.md

```diff
@@ -0,0 +1,154 @@
+Below is a minimal example webpart that demonstrates how to use MGT with disambiguation in SharePoint Framework Webparts. A more complete example is available in the [No Framework Webpart Sample](../../samples/sp-mgt/src/webparts/helloWorld/HelloWorldWebPart.ts).
+
+```ts
+import { BaseClientSideWebPart } from '@microsoft/sp-webpart-base';
+import { Providers } from '@microsoft/mgt-element';
+import { SharePointProvider } from '@microsoft/mgt-sharepoint-provider';
+import { customElementHelper } from '@microsoft/mgt-element/dist/es6/components/customElementHelper';
+import { importMgtComponentsLibrary } from '@microsoft/mgt-spfx-utils';;
+
+export default class MgtWebPart extends BaseClientSideWebPart<Record<string, unknown>> {
+  private _hasImportedMgtScripts = false;
+  private _errorMessage = '';
+
+  protected onInit(): Promise<void> {
+    if (!Providers.globalProvider) {
+      Providers.globalProvider = new SharePointProvider(this.context);
+    }
+    customElementHelper.withDisambiguation('foo');
+    return super.onInit();
+  }
+
+  private onScriptsLoadedSuccessfully() {
+    this.render();
+  }
+
+  public render(): void {
+    importMgtComponentsLibrary(this._hasImportedMgtScripts, this.onScriptsLoadedSuccessfully, this.setErrorMessage);
+
+    this.domElement.innerHTML = `
+    <section class="${styles.helloWorld} ${this.context.sdks.microsoftTeams ? styles.teams : ''}">
+      ${this._renderMgtComponents()}
+      ${this._renderErrorMessage()}
+    </section>`;
+  }
+
+  private _renderMgtComponents(): string {
+    return this._hasImportedMgtScripts
+      ? '<mgt-foo-login></mgt-foo-login>'
+      : '';
+  }
+
+  private setErrorMessage(e?: Error): void {
+    if (e) this.renderError(e);
+
+    this._errorMessage = 'An error ocurred loading MGT scripts';
+    this.render();
+  }
+
+  private _renderErrorMessage(): string {
+    return this._errorMessage
+      ? `<span>${this._errorMessage}</span>`
+      : '';
+  }
+}
+```
+
+### When using React to build webparts
+
+When building SharePoint Framework web parts using React any component that imports from the `@microsoft/mgt-react` library must be asynchronously loaded after configuring the disambiguation setting. The `lazyLoadComponent` helper function exists to facilitate using `React.lazy` and `React.Suspense` to lazy load these components from the top level webpart.
+Below is a minimal example webpart that demonstrates how to use MGT with disambiguation in React based SharePoint Framework Webparts. A complete example is available in the [React SharePoint Webpart Sample](../../samples/sp-webpart/src/webparts/mgtDemo/MgtDemoWebPart.ts).
```

Comment:
This TS file won't work great on npmjs.org I think. I think we should fully qualify the github URL.

Reply:
Hmm, shall we point it at the url where it will be when we merge this to main, or the next/fluentui branch and update when v3 ships to main?

Reply:
I feel this should be main